apps/src/makerlab/PlaygroundComponents.js

```diff
@@ -74,7 +79,7 @@ export function initializeCircuitPlaygroundComponents(io, five, PlaygroundIO) {
     addSensorFeatures(five.Board.fmap, s);
   });
 
-  return _.assign({}, touchSensors, {
+  return Object.assign({}, touchPads, {
```

Comment:
Could also do this using object destructuring (which has become my general preference)
```
return {
  ...touchPads,
  colorLeds: colorLeds,
  // etc
}